test/cql-pytest/rest_api.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# This file provides utility to REST API requests.
+# Some metrics cannot be obtained by nodetool, but they are available by API.
+
+import requests
+import nodetool
+import pytest
+
+# Sends GET request to REST API. Response is restured as JSON.
+# If API isn't available, `pytest.skip()` is called.
+def get_request(cql, *path):
+    if nodetool.has_rest_api(cql):
+        response = requests.get(f"{nodetool.rest_api_url(cql)}/{'/'.join(path)}")
+        return response.json()
+    else:
+        pytest.skip("REST API not available")
+
+# Get column family's metric. Optional argument `table` is column family name 
+# parameter in API's endpoint formatted like `keyspace:name`.
```

Comment:
‎This last sentence is confusing: According to the code below the "table" parameter is in fact expected to be formatted as "keyspace.name", *not* "keyspace:name".

Reply:
You are right, format with `:` is expected by API, function expects format with '.' (this format is returned by  python functions, e.g. `new_test_table`). I hope the description is more clear now.